spotbugs/src/main/java/edu/umd/cs/findbugs/SAXBugCollectionHandler.java

```diff
@@ -573,7 +578,38 @@ private void parseBugInstanceContents(String qName, Attributes attributes) throw
         } else if ("UserAnnotation".equals(qName)) {
             // legacy from the cloud plugin stuff
         } else {
-            throw new SAXException("Unknown bug annotation named " + qName);
+            // @Anemone, add custom bug annotation types,
+            // which can be deserialized with 'fromXML' method in its class.
+            Method fromXML = qnameCache.computeIfAbsent(qName, k -> {
+                for (Plugin plugin : Plugin.getAllPlugins()) {
+                    Class<?> annotationClazz;
+                    try {
+                        // The qName should equal to its classname, so there can reflect.
```

Comment:
What is "there can reflect" meant to say?

Reply:
We use annotation's name as qName, so we can reflect the class by 'qName'. I also modified the comment in code.